packages/admin-panel-server/examples.http

```diff
@@ -0,0 +1,113 @@
+@baseUrl = http://localhost:8070/v1
+# Uncomment below to fetch against the dev server, rather than localhost
+# @baseUrl = https://dev-admin-api.tupaia.org/v1
+@contentType = application/json
+# In order to setup authorization, please set 'email' and 'password' in your restClient environement variables
+# see: https://marketplace.visualstudio.com/items?itemName=humao.rest-client#environment-variables
+@authorization  = Basic {{email}}:{{password}}
+
+### Fetch Dashboard Preview Data
+POST {{baseUrl}}/fetchReportPreviewData?entityCode=TO
```

Comment:
We do need to send in the hierarchy here as well, however I think we agree to scrap the hierarchy selector and use the 'Project' selector that sits near the header, right @tcaiger ? If so I guess the hierarchy will be part of the 'main' config of the viz, so we will need that to come in this payload somehow.

There's actually another sub issue here, tho we may be able to avoid it for now, but 'Projects' and hierarchies aren't actually the same thing... We are _**meant**_ to be able to share one hierarchy among many projects, and thus not have a one-to-one mapping between projects and hierarchies, however we've never properly implemented this as yet. For now, I believe that there's always a hierarchy for a project, and they always share the same code as well (may need to double check me on that one)? If so, then project code should suffice, but we should be aware that this may change in the future...

Reply:
Yeah I was aware of this, and I thought we can actually temporarily solve this by letting the /projects endpoint in meditrak-server to also return the entity hierarchy code (we just need to do a custom join). Might not be the best way but I think it's the safer approach than assuming project code is also the same as the entity hierarchy code/name